app/Connector/packages/connector-theme/sass/var/view/_Learn.scss

```diff
@@ -872,9 +872,22 @@ img.detail-has-data-small {
   }
 
   .detail-row {
-    height: 123px;
+    height: 125px;
+
     .detail-description {
-      height: 74px;
+      height: 105px;
+
+      .detail-description-text {
+        padding-top: 12px;
+        font: $body-one-font;
+        color: $heat-scale-1;
+        margin-right: 5%;
+        text-align:initial;
+        overflow-wrap: break-word;
```

Comment:
Is break-word acceptable per DataSpace design? Also, for Antigen name, I thought the requirement is to not break a component (which contains words).

Reply:
Now removed, initially I thought it was required, but it is not necessary, esp. since i've implemented the component wrapping requirement differently.